### PR TITLE
Fix: operation signatories wallet and operation details order

### PR DESCRIPTION
**Pull request by egor0798**
## Changes
- added `signatoryUtils` with `getSignatoryWallet`
- Fixed operation details field order and depositor wallet

### CODE REVIEW

**Comment by egor0798** [comment]
Task linked: [CU-8692zj3wr Operation details. Data display problems](https://app.clickup.com/t/8692zj3wr)

**Comment by egor0798** [comment]
Task linked: [CU-8692zj2ck Operations. Wrong signatories definition](https://app.clickup.com/t/8692zj2ck)

File: src/renderer/entities/signatory/lib/utils.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import keyBy from 'lodash/keyBy';
+
+import { Account, AccountId, Wallet } from '@renderer/shared/core';
+import { walletUtils } from '@renderer/entities/wallet';
+
+export const singnatoryUtils = {
+  getSignatoryWallet,
+};
+
+function getSignatoryWallet(accoutId: AccountId, accounts: Account[], wallets: Wallet[]): Wallet | undefined {
```
Comment (tuul-wq) [inline src/renderer/entities/signatory/lib/utils.ts:10]: More meaningful parameters order, 2 fast returns
please check `return walletsMap[signatoryAccount?.walletId];`, I guess it should work 🤔
```javascript
function getSignatoryWallet(wallets: Wallet[], accounts: Account[], accoutId: AccountId): Wallet | undefined {
  const walletsMap = keyBy(wallets, 'id');

  const signatoryAccount = accounts.find((account) => {
    const accountIdMatch = accoutId === account.accountId;
    const wallet = walletsMap[account.walletId];
    
    if (!accountIdMatch || !wallet) return;

    return walletUtils.isValidSignatory(wallet) || walletUtils.isMultiShard(wallet);
  });

  return walletsMap[signatoryAccount?.walletId];
}
```

**Review by tuul-wq — APPROVED** [review]
Nice

**Review by Asmadek — APPROVED** [review]

File: src/renderer/entities/signatory/lib/utils.ts (hunk 1 above)
Comment (egor0798) [reply src/renderer/entities/signatory/lib/utils.ts:10]: `return walletsMap[signatoryAccount?.walletId]`
typescript error - cant use undefimed as index type

**Pull request merged**